test/configuration_test.dart

```diff
@@ -174,16 +174,6 @@ Future<void> main([List<String>? args]) async {
     expect(cars.length, 1);
   });
 
-  test('Configuration readOnly - writing on read-only Realms throws', () {
```

Comment:
Is this test not needed anymore and why is that?

Reply:
It is moved to realm_test.dart in order to be able to use `_writeAndExpectException`, since you wanted `_writeAndExpectException` to be private.